python/docs/source/getting_started/quickstart_ps.ipynb

```diff
@@ -99,89 +99,21 @@
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/html": [
```

Comment:
Actually can you also manually fix the HTML here instead of using `print(psdf)`?

Reply:
If that's possible, it would really be great :-).

Reply:
Okay, I'll give it a try.